zello-sdk/src/com/zello/sdk/SafeHandler.java

```diff
@@ -1,18 +1,20 @@
 package com.zello.sdk;
 
+import android.content.Context;
 import android.os.Handler;
+import android.os.Looper;
 import android.os.Message;
 
 import java.lang.ref.WeakReference;
 
 import androidx.annotation.NonNull;
 
-@SuppressWarnings({"WeakerAccess", "unused"})
 class SafeHandler<T extends SafeHandlerEvents> extends Handler {
 
 	private final @NonNull WeakReference<T> _t;
 
-	public SafeHandler(T t) {
+	public SafeHandler(@NonNull T t, @NonNull Context context) {
+		super(context.getMainLooper(), null);

```

Comment:
Another minor one -- I'm pretty sure `Handler` has a single-parameter constructor that accepts just a `Looper`. Unless passing null here does something specific?

Reply:
You are totally right. I think the two parameter constructor got in there when I was trying to avoid having to pass in a looper while I was attempting to get rid of a warning.